docs/auth/auth0/provider.md

```diff
@@ -37,6 +37,8 @@ auth:
         audience: ${AUTH_AUTH0_AUDIENCE}
         connection: ${AUTH_AUTH0_CONNECTION}
         connectionScope: ${AUTH_AUTH0_CONNECTION_SCOPE}
+  session:
+    secret: 'supersecret'
```

Comment:
I think it's best to make this an env var as well, to make sure that people fill it in with their own secret. Probably `AUTH_SESSION_SECRET`

Reply:
Sure